daft/series.py

```diff
@@ -635,6 +635,12 @@ def left(self, pattern: Series) -> Series:
         assert self._series is not None and pattern._series is not None
         return Series._from_pyseries(self._series.utf8_left(pattern._series))
 
+    def right(self, pattern: Series) -> Series:
+        if not isinstance(pattern, Series):
+            raise ValueError(f"expected another Series but got {type(pattern)}")
+        assert self._series is not None and pattern._series is not None
+        return Series._from_pyseries(self._series.utf8_right(pattern._series))
```

Comment:
```suggestion
    def right(self, nchars: Series) -> Series:
        if not isinstance(nchars, Series):
            raise ValueError(f"expected another Series but got {type(nchars)}")
        assert self._series is not None and nchars._series is not None
        return Series._from_pyseries(self._series.utf8_right(nchars._series))
```
also for the `left` function above this one if you don't mind!